util/statvar_dcid_generator.py

```diff
@@ -458,10 +492,12 @@ def _process_constraint_property(prop: str, value: str) -> str:
     Returns:
         A string that can be used in dcid generation.
     """
-    if prop == 'naics':
+    if 'NAICS' in value:
```

Comment:
perhaps "startswith" here? could NAICS appear elsewhere in the name?

Reply:
There is a case where NAICS could appear elsewhere in the name.
The value could also be prepended by a namespace like `dcs: NAICS/11`.

Reply:
sounds good